### PR TITLE
add grant testing faqs

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/301986105

Changes proposed in this pull request:

- https://pr-7415-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/faq/testing-faq/
- info and source doc in b/301986105
-

### CODE REVIEW

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Starting in Q1 2024, Chrome will deprecate third-party cookies on 1% of traffic and this is referred to as the Mode B Chrome-facilitated testing traffic in Chrome developer documentation. In a small fraction of Mode B Chrome-facilitated testing traffic, Privacy Sandbox technologies will also not be available in addition to third-party cookies being deprecated. Using a combination of all of the above Chrome-facilitated testing modes, ad techs will be able to align their test setups to the testing guidance published by the CMA. (Refer to [Chrome-facilitated testing](/docs/privacy-sandbox/chrome-testing/#:~:text=Mode%20A%3A%20Ad%20techs%20can,portion%20of%20all%20Chrome%20users.) which defines Mode A and Mode B and [CMA Guidance](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf) defines Control 1, Control 2, Treatment groups (in point number 11)
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:45]: Formatting of the last sentence is a bit off—a bracket opened and not closed, a bracket within a bracket, no full-stop at the end et cetera

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    -   What SSP's `reportResult()` will make available to the DSP's `reportWin()`.
+-   **Post-auction reporting**: DSP to SSP. Context: The DSP must capture and delegate post-auction reports to the SSP. If the SSP needs clicks, views, viewability metrics, the DSP must enable them to receive these events.
+    -   What reports the SSP will need.
+    -   The methodology for measuring that event (such as, viewability definition).
+    -   DSP implementation of the `reportEvent()` call to support the SSP requirements.
+    -   SSP implementation of the `registerAdBeacon()` lining up the event naming with what the DSP will trigger in the creative.
+
+**Are the Attribution Reporting API and the Aggregation Service ready and available for testing?**
+
+The Attribution Reporting API will be generally available and ramp-up is already in progress. With that, Attribution Reporting will soon be available for use on 100% of traffic. Note that Attribution Reporting can be used with all ads, not just those served by Protected Audience.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:35]: I feel like if a question asks about the Aggregation Service, the response should at least mention it once, even if the assumption is people understand the context.

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+**Is there a contact at the CMA we could connect with to better understand what is required for the submitted final report?**
+
+You can email the CMA case team at <a href="mailto:privacysandbox@cma.gov.uk">privacysandbox@cma.gov.uk</a> and read more about their testing proposal guidance [here](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). (For additional reference, the CMA lists all their contacts at the bottom of their [Privacy Sandbox page](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes))
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:49]: Remove link from [here] and instead add descriptive link text. This sentence too has unnecessary brackets imo

Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:49]:
```suggestion
You can email the CMA case team at <a href="mailto:privacysandbox@cma.gov.uk">privacysandbox@cma.gov.uk</a> and read more about their testing proposal guidance [here](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). (For additional reference, the CMA lists all their contacts at the bottom of their [Privacy Sandbox page](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes)).
```

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+**Could you please clarify quantitative testing success metrics?**
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:51]:
```suggestion
**Can you clarify the quantitative testing success metrics?**
```

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+Grant participants can define and share as many additional KPIs as they want, beyond those specified in the SOW. The minimum test results that need to be shared are listed in the [CMA Guidance](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). This guidance is purposefully "light" and broad so the CMA can compare many results from different ad techs.
+
+**Is there a volume threshold for testing? (Control group ratios vs. Modes ratios, etc.) Do you have additional guidance on valuable KPIs to test beyond the CMA's testing guidance?**
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:55]:
```suggestion
**Is there a volume threshold for testing, for example, control group ratios versus modes ratios? Do you have additional guidance on valuable KPIs to test beyond the CMA's testing guidance?**
```

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+You can draft these volume/scale requirements in your Integration Plan; we will work with you to align on the scale required given your business model. You can include specific metrics and tests in your Testing Plan. Please keep in mind that the Testing Plan should take into account Mode A and Mode B of Chrome-facilitated testing. We can provide feedback on these once you submit them.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/faq/testing-faq/index.md:57]:
```suggestion
You can draft these volume / scale requirements in your integration plan. We will work with you to align on the scale required given your business model. You can include specific metrics and tests in your testing plan. Please keep in mind that the testing plan should take into account Mode A and Mode B of Chrome-facilitated testing. We can provide feedback on these once you submit them.
```

File: site/en/docs/privacy-sandbox/faq/testing-faq/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/faq/testing-faq/index.md:35]: Good catch.

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/faq/testing-faq/index.md:35]: Asking now

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/faq/testing-faq/index.md:35]: still waiting. Might get answer today.

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**